From: Ruben Undheim <ruben.undheim@gmail.com>
Date: Sun, 5 Aug 2018 19:28:21 +0000
Subject: Fix build

---
 setup.py | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/setup.py b/setup.py
index a3d05a8..43c0732 100644
--- a/setup.py
+++ b/setup.py
@@ -16,13 +16,14 @@ except ImportError:
 
 
 def read(filename):
-    with open(filename, 'r') as f:
+    with open(filename, 'rb') as f:
         return f.read()
 
+a2 = read('AUTHORS').decode('utf-8')
+a3 = read('CHANGES').decode('utf-8')
+a1 = read('README').decode('utf-8')
 
-long_description = '\n\n'.join([read('README'),
-                                read('AUTHORS'),
-                                read('CHANGES')])
+long_description = '\n\n'.join([a1,a2,a3])
 
 __doc__ = long_description
 
