Index: pyvisa/setup.py
===================================================================
--- pyvisa.orig/setup.py
+++ pyvisa/setup.py
@@ -4,4 +4,11 @@
 from setuptools import setup
 
 if __name__ == "__main__":
-    setup(use_scm_version=True)
+    setup(use_scm_version=True, packages=[
+        'pyvisa',
+        'pyvisa.ctwrapper',
+        'pyvisa.resources',
+        'pyvisa.testsuite',
+        'pyvisa.testsuite.keysight_assisted_tests',
+        'pyvisa.thirdparty',
+        ])
